UI_UX/Frontend/src/components/ExchangeRate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+
+//value = -1 => loading state
+export const ExchangeRate = ({ value }) => {
+  let loadingIcon = undefined;

```

Comment:
Again, no unnecessary `undefined` clauses, use `const` and define only when needed, by default all unassigned variables are `undefined`

Reply:
fixed